CHANGELOG.md

```diff
@@ -1,3 +1,22 @@
+# Release 0.26.9
+
+## What's New
+N/A
+
+## Ziti Library Updates / Bug Fixes
+
```

Comment:
Bug in the release notes tool. It doesn't account for the /v2 at the end. I'll have to fix that. Not a big deal for now.